src/styles/blocks/product-card.scss

```diff
@@ -0,0 +1,97 @@
+.product-card {
+  box-sizing: border-box;
+  width: 200px;
+  padding: 32px 16px 16px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+
+  @include animation-hover(transform, scale(1.2));
```

Comment:
The mixin 'animation-hover' is not defined in this file or the provided files. Make sure that the mixin exists and is imported correctly according to the [Code Style Rules ❗️](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules).